shared/src/main/kotlin/com/egm/stellio/shared/model/EntitiesEvent.kt

```diff
@@ -0,0 +1,97 @@
+package com.egm.stellio.shared.model
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.fasterxml.jackson.annotation.JsonSubTypes
+import com.fasterxml.jackson.annotation.JsonTypeInfo
+import com.fasterxml.jackson.annotation.JsonTypeName
+import java.net.URI
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.EXISTING_PROPERTY, property = "operationType")
+@JsonSubTypes(
+    *[
+        JsonSubTypes.Type(value = EntityCreateEvent::class),
+        JsonSubTypes.Type(value = EntityDeleteEvent::class),
+        JsonSubTypes.Type(value = AttributeAppendEvent::class),
+        JsonSubTypes.Type(value = AttributeReplaceEvent::class),
+        JsonSubTypes.Type(value = AttributeUpdateEvent::class),
+        JsonSubTypes.Type(value = AttributeDeleteEvent::class)
+    ]
+)
+open class EntitiesEvent(
+    val operationType: EventsType,
+    open val entityId: URI
+) {
+    @JsonIgnore
+    open fun getEntity(): String? = null
+    @JsonIgnore
+    open fun getEventPayload(): String? = null
```

Comment:
by looking at current code, it seems useless since it always returns the operation payload